assets/training/finetune_acft_image/components/finetune/mmt/spec.yaml

```diff
@@ -0,0 +1,341 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
```

Comment:
Would you need any changes in the supporting finetune.py file?
If so, are you planning on pulling in those at a later point in time?

Reply:
added. ;) thank you.